docs/docs/run_minder_server/installing_minder.md

```diff
@@ -0,0 +1,81 @@
+# Installing Minder
```

Comment:
I think this should be "Installing Minder with Helm" or something like that, given the other topics.

You may also want to add some front-matter, like the other docs, e.g.:

```
---
sidebar_label: Helm Install
sidebar_position: 80
---
```